### DISCUSSION TITLE
https://fruition-with-comments.shj.rip/e2fb22e260344f70b285db1e5572acb3

### DISCUSSION THREAD

**Discussion by giscus** [Gumball12/fruition-with-comments#1] [General]
# https://fruition-with-comments.shj.rip/e2fb22e260344f70b285db1e5572acb3

A new tool that blends your everyday work apps into one. It's the all-in-one workspace for you and your team

https://fruition-with-comments.shj.rip/e2fb22e260344f70b285db1e5572acb3

**Comment by Gumball12** [comment]
Hello from **fruition-with-comments**!
([you can see this comment at this link](https://github.com/Gumball12/fruition-with-comments/discussions/1))

**Comment by zai1zhi** [comment]
hi, it is awesome! 
I followed these steps but it didn't work (the Cloudflare Worker script can save and deploy). It just has not been added to my notion...

**Reply by Gumball12** [reply]
Hi. I think it might be because the Fruition code changed.
If you don't have to hurry, I'll check in a minute. Thank you!
